src/stores/muutospyynnotEsittelija.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, createHook } from "react-sweet-state";
+import { execute } from "./utils/loadFromBackend";
+
+const Store = createStore({
+  initialState: {},
+  actions: {
+    load: statusKey => ({ getState, setState }) => {
+      return execute(
+        { getState, setState },
+        {
+          key: "muutospyynnot",
+          urlEnding: statusKey //avoimet, valmistelussa, paatetyt
+        }
+      );
+    }
+  },
+  name: "Muutospyynnöt"
```

Comment:
stores/muutospyynnot.js is already using the same name. Could this be e.g. "Muutospyynnöt - Esittelijä"?